mmv1/third_party/terraform/tests/resource_container_node_pool_test.go.erb

```diff
@@ -156,6 +156,124 @@ func TestAccContainerNodePool_noName(t *testing.T) {
 	})
 }
 
+func TestAccContainerNodePool_withNoSpecifiedLoggingVariant(t *testing.T) {
```

Comment:
We probably don't need this test - it effectively is checking that terraform field Defaults work properly, which we don't need to be checking in these tests - especially in container cluster tests since there's a startup cost for each test.

Reply:
Ack.